modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -1849,6 +1849,8 @@ else if (ttl != CU.TTL_ZERO)
             if (updateRes.success())
                 updateMetrics(c.op, metrics);
 
+            Map<UUID, CacheContinuousQueryListener> curLsnrs;
```

Comment:
May be we need to move this declaration inside else block?
Outside that block variable never used.